pages/common/pabcnetcclear.md

```diff
@@ -0,0 +1,12 @@
+# pabcnetcclear
```

Comment:
Wondering if this should go in the `windows` pages instead of common ?
As per the [website/downloads](http://pascalabc.net/en/download), seems its only for windows.

Reply:
PascalABC.NET also runs under mono [on Linux](http://pascalabc.net/wiki/index.php?title=%D0%9A%D0%B0%D0%BA_%D0%B8%D0%BD%D1%81%D1%82%D0%B0%D0%BB%D0%BB%D0%B8%D1%80%D0%BE%D0%B2%D0%B0%D1%82%D1%8C_PascalABC.NET_%D0%BF%D0%BE%D0%B4_Linux).

Reply:
I think so too. If it's primarily designed for Windows, it should be in the `windows/` directory.

Reply:
Done.